test/unit/lodash.spec.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function()
+{
+
+    describe('constant', function()
+    {
+        var params = datasets.constant();
+        it('should check if first parameter is an object',function()
+        {
+            expect(params[0]).to.be.an('object');
+        });
+        it('should check objects values',function()
+        {
+            expect(params[0]).to.have.property('fred');
+            expect(params[0]).to.have.property('pebbles');
+            expect(params[0]).to.have.deep.property('john.age', 36);
+        });
+        it('should check if function returns an object', function(){
+             var getter = _.constant(params[0]);
+             expect(getter()).to.be.an('object');
+        });
+        it('should create a function that returns value', function(){
+                expect(_.constant.apply(_, params)).to.be.a('function');
+        });
+    });
+
+    describe('conforms', function()
+    {
+        var params = datasets.conforms();
+        var fun = _.conforms({ 'number': _.partial(_.gt, _, 38) });
```

Comment:
Here you named variable which is result of lodash function `fun`, in other place `getter`. Use one convention in every tests.